src/Makefile.am

```diff
@@ -67,5 +67,11 @@ libseccomp_la_CFLAGS = ${AM_CFLAGS} ${CODE_COVERAGE_CFLAGS} ${CFLAGS} \
 libseccomp_la_LDFLAGS = ${AM_LDFLAGS} ${CODE_COVERAGE_LDFLAGS} ${LDFLAGS} \
 	-version-number ${VERSION_MAJOR}:${VERSION_MINOR}:${VERSION_MICRO}
 
+EXTRA_DIST += arch-x86_64-syscalls.c
+CLEANFILES = arch-x86_64-syscalls.c
+
+arch-x86_64-syscalls.c: arch-x86_64-syscalls.c.perf
+	$(GPERF) -m 100 --null-strings --pic -ptCEG -S1 $< > $@
```

Comment:
what is -p?  I don't see it in my help for gperf

Reply:
good eye!  I assumed it was used by older versions (https://git.savannah.gnu.org/cgit/gperf.git/tree/src/options.cc#n934):
```
        case 'p':               /* Generated lookup function a pointer instead of int. */
          break;                /* This is now the default. */
```

but it is a no-op in all the releases and as far as the GNU perf git history goes, so I'll drop it